log.h

```diff
@@ -30,7 +30,7 @@
 #include "pantavisor.h"
 #include <stdarg.h>
 
-void exit_error(int err, char *msg);
+void exit_error(int err, char *msg) __attribute__((__noreturn__));
```

Comment:
FYI: this it seems to be mandatory for this kind of function on the clang-tidy are called [custom assertion handlers](https://clang-analyzer.llvm.org/annotations.html#custom_assertions)

Reply:
I don't understand this very well. Does it mean all void returning functions should have this?

Reply:
Not all, only those finish the execution. It seems clang-tidy sometimes has problems to recognize when a function is a "custom assertion handler"